main.ts

```diff
@@ -106,7 +127,7 @@ export default class Review extends Plugin {
 		let parsedResult = naturalLanguageDates.parseDate(someDate);
 		let inputDate = parsedResult.formattedString;
 
-		console.log("Date string to use: " + inputDate);
+		console.debug("Date string to use: " + inputDate);

```

Comment:
I just switched these to debug so that they don't pollute the console as much, but it might make sense to remove some of these console statements still!